### PR TITLE
BaSP-MR-58: Add and edit trainers form

### PR DESCRIPTION
**Pull request by EmaBaldor**
Added add and edit trainers form. 
I worked in the trainers list and added new functions of edit and added new trainer.

### CODE REVIEW

File: src/Components/Trainers/Form/index.js
Comment (AleMac17) [inline src/Components/Trainers/Form/index.js:1]: We are using the jsx extension

File: src/Components/Trainers/Modal/index.js
Comment (AleMac17) [inline src/Components/Trainers/Modal/index.js:1]: We are using the jsx extension

File: src/Components/Trainers/Table/index.js
Comment (AleMac17) [inline src/Components/Trainers/Table/index.js:1]: We are using the jsx extension

**Review by AleMac17 — CHANGES_REQUESTED** [review]

File: src/Components/Trainers/index.jsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        <button onClick={handleCloseModal}>Close</button>
+      </div>
+    );
+  };
+  const deleteItem = async (id) => {
+    try {
+      await fetch(process.env.REACT_APP_API_URL + `/trainers/${id}`, {
+        method: 'DELETE'
+      });
+
```
Comment (AleMac17) [inline src/Components/Trainers/index.jsx]: We should not leave empty lines

File: src/Components/Trainers/trainers.module.css [hunk 2]
```diff
@@ -1,6 +1,49 @@
...
+  width: 120px;
+  background-color: #fff;
+  color: #000000;
+  border: #000000 1px solid;
+  border-radius: 5px;
+  padding: 5px 10px;
+  cursor: pointer;
+  margin: 1rem;
+  cursor: pointer;
+}
```
Comment (AleMac17) [inline src/Components/Trainers/trainers.module.css:25]: We should leave empty lines between css styles

File: src/Components/Trainers/trainers.module.css [hunk 3]
```diff
@@ -1,6 +1,49 @@
...
-    align-items: center;
-  }
\ No newline at end of file
+  flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  gap: 0.5rem;
+}
```
Comment (AleMac17) [inline src/Components/Trainers/trainers.module.css:8]: We should leave empty lines between css styles

File: src/Components/Trainers/Form/index.js
Comment (EmaBaldor) [reply src/Components/Trainers/Form/index.js:1]: Changed extensions.

File: src/Components/Trainers/Modal/index.js
Comment (EmaBaldor) [reply src/Components/Trainers/Modal/index.js:1]: Changed extensions.

File: src/Components/Trainers/Table/index.js
Comment (EmaBaldor) [reply src/Components/Trainers/Table/index.js:1]: Changed extensions.

**Comment by EmaBaldor** [comment]
[[Trainer] Add/Edit Form](https://trello.com/c/iv8Uyw4L/58-trainer-add-edit-form)

File: src/Components/Trainers/index.jsx (hunk 1 above)
Comment (EmaBaldor) [reply src/Components/Trainers/index.jsx]: Resolved.

File: src/Components/Trainers/trainers.module.css (hunk 2 above)
Comment (EmaBaldor) [reply src/Components/Trainers/trainers.module.css:25]: Resolved.

File: src/Components/Trainers/trainers.module.css (hunk 3 above)
Comment (EmaBaldor) [reply src/Components/Trainers/trainers.module.css:8]: Resolved.

**Review by AleMac17 — APPROVED** [review]

**Review by lorenzomauro59 — APPROVED** [review]
For me, It's approved!

File: src/Components/Trainers/index.jsx [hunk 4]
```diff
@@ -1,58 +1,139 @@
...
+  const flagForm = () => {
+    setShowForm(!showForm);
+    setEditMode(false);
+    setSelectedTrainer(null);
   };
   useEffect(() => {
     getTrainers();
   }, []);
-  const deleteModal = () => {
+  const Modal = () => {
```
Comment (LucianoAlarcon) [inline src/Components/Trainers/index.jsx]: Please change the name to `modal`

**Review by LucianoAlarcon — CHANGES_REQUESTED** [review]

File: src/Components/Trainers/index.jsx (hunk 4 above)
Comment (EmaBaldor) [reply src/Components/Trainers/index.jsx]: Resolved.

**Review by LucianoAlarcon — APPROVED** [review]

**Pull request merged**